source/backend/apimodels/Models/Concepts/AcquisitionFile/AcquisitionFileModel.cs

```diff
@@ -96,7 +96,7 @@ public class AcquisitionFileModel : FileWithChecklistModel
         /// <summary>
         /// get/set - A list of research property relationships.
         /// </summary>
-        public IList<AcquisitionFilePropertyModel> FileProperties { get; set; }
+        public new IList<AcquisitionFilePropertyModel> FileProperties { get; set; }
```

Comment:
I'm not familiar with this syntax. Why is it only being applied to this one list?